compiler/test/test_wasm_utils.re

```diff
@@ -80,18 +70,18 @@ let test_leb128_i32 = ctxt => {
 };
 
 let test_get_wasm_sections = ctxt => {
-  ignore(compile_wasm_file("testmod"));
   let inchan = open_in_bin("test-data/testmod.wasm");
   let sections = get_wasm_sections(inchan);
   close_in(inchan);
   assert_equal_sections(
     ~ctxt,
     [
-      {sec_type: Type, offset: 14, size: 8},
-      {sec_type: Import, offset: 28, size: 25},
-      {sec_type: Function, offset: 59, size: 2},
-      {sec_type: Export, offset: 67, size: 17},
-      {sec_type: Code, offset: 90, size: 12},
+      {sec_type: Type, offset: 10, size: 8},
+      {sec_type: Import, offset: 20, size: 25},
+      {sec_type: Function, offset: 47, size: 2},
+      {sec_type: Export, offset: 51, size: 17},
+      {sec_type: Code, offset: 70, size: 8},
+      {sec_type: Custom("name"), offset: 85, size: 28},
```

Comment:
Just curious why these offsets changed

Reply:
Just a different compiler. The old one was compiled via (a pretty old version of) the wasm spec, versus this one was compiled via wat2wasm. I figured we should check in the one compiled with wat2wasm since we won't really have easy access to generating the `.wasm` file again via the spec if we need to, but someone's probably got [the wabt suite](https://github.com/WebAssembly/wabt) on their machine (which includes wat2wasm).